fairscale/optim/adascale.py

```diff
@@ -509,7 +517,12 @@ def step(self, *args: Any, **kwargs: Any) -> Optional[float]:
         original_lr = []
         for idx, param_group in enumerate(self._optimizer.param_groups):
             original_lr.append(param_group["lr"])
-            param_group["lr"] = self.gain(pg_idx=idx) * param_group["lr"]
+            if self._num_grads_to_accum > 1 and not self._is_scaled_loss:
+                # Divide by num_grads_to_accum to account for gradient accumulation.
```

Comment:
this comment only states the same info as the code, which is not very useful. Let's explain that when scaled loss is not used, the gain computed is correct, but in order to match what DDP does, we need to scale the grad back by N.

However, with momentum, scale LR by N here is not the same as scale the param.grad by N in the DDP case, right?